web/packages/teleport/src/Navigation/SideNavigation/Navigation.tsx

```diff
@@ -86,6 +88,14 @@ export type NavigationSubsection = {
   icon: (props) => JSX.Element;
   parent?: TeleportFeature;
   searchableTags?: string[];
+  /* customRouteMatchFn is a custom function for determining whether this subsection is currently active,
+  this is useful in cases where a simple base route match isn't sufficient. */
```

Comment:
Nit: it is not a valid JSDoc format (here and in other places).
```suggestion
  /**
   * customRouteMatchFn is a custom function for determining whether this subsection is currently active,
   * this is useful in cases where a simple base route match isn't sufficient.
   */
```

Reply:
I also meant that each line should start with `*` (it improves readability too).